content/tracing/advanced_usage/custom_tagging.md

```diff
@@ -46,6 +46,34 @@ class ServletImpl extends AbstractHttpServlet {
 {{% tab "Python" %}}
 {{% /tab %}}
 {{% tab "Ruby" %}}
+**Adding tags to a span**
```

Comment:
I think we should add also an example to retrieve the current active span, otherwise people don't know how to retrieve the `active_span`.

Reply:
Addressed in 
https://github.com/DataDog/documentation/pull/3025/commits/3d9d13f8a6cccb2d842e7160237d4ffe0659f514